### PR TITLE
Add support for Coveralls reporting

### PR DESCRIPTION
**Pull request by hummusonrails**
Adding Coveralls reporting coverage to the gem

### CODE REVIEW

**Comment by hummusonrails** [comment]
@timcraft - Any idea why the jruby builds continue to fail until I go in manually and restart the builds? It looks like it's failing on the downloads.

**Comment by hummusonrails** [comment]
> @timcraft - Any idea why the jruby builds continue to fail until I go in manually and restart the builds? It looks like it's failing on the downloads.

This seems to be a new issue with jruby, and not with us, as @fabianrbz discovered, FYI.

**Comment by timcraft** [comment]
> Any idea why the jruby builds continue to fail until I go in manually and restart the builds? It looks like it's failing on the downloads.

I noticed it before but no idea. The call that is failing is `rvm get stable` so even though it's only on the jruby builds I don't think it's a jruby issue. Whether it's an rvm issue, a travis issue, or an issue with the travis config file we're using I'm not sure 🤷‍♂️

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Nexmo Client Library for Ruby
 
-[![Gem Version](https://badge.fury.io/rb/nexmo.svg)](https://badge.fury.io/rb/nexmo) [![Build Status](https://api.travis-ci.org/Nexmo/nexmo-ruby.svg?branch=master)](https://travis-ci.org/Nexmo/nexmo-ruby)
+[![Gem Version](https://badge.fury.io/rb/nexmo.svg)](https://badge.fury.io/rb/nexmo) [![Build Status](https://api.travis-ci.org/Nexmo/nexmo-ruby.svg?branch=master)](https://travis-ci.org/Nexmo/nexmo-ruby) [![Coverage Status](https://coveralls.io/repos/github/Nexmo/nexmo-ruby/badge.svg?branch=coveralls)](https://coveralls.io/github/Nexmo/nexmo-ruby?branch=coveralls)
```
Comment (fabianrbz) [inline README.md]: is there a way to pick up the current branch? if not we should change it to master.

**Review by fabianrbz — CHANGES_REQUESTED** [review]
@benhayehudi looks good! although i left a comment about the branch, it should point to master right?

File: README.md (hunk 1 above)
Comment (hummusonrails) [reply README.md]: Yes, good catch.

**Pull request merged**